en/docs/reference/customize-product/customizations/customizing-the-developer-portal/enabling-or-disabling-self-signup.md

```diff
@@ -45,7 +45,7 @@ In a multi-tenanted API Manager setup, self-signup to the Developer Portal works
 
 5. Sign in to the admin portal  ( `https://<HostName>:9443/admin` ) as tenant admin and navigate to **Advance Configurations** by clicking **Advance** under **Settings**.
 
-6. Add the following configuration and save the content.
+6. Add the following configuration and save the content. This was added because even though the new user can create a new account, that new user can't login to devportal without the `Internal/subscriber` role.
```

Comment:
Let's reword this as follows.

The `SignUpRoles` configuration indicates the default roles that will be assigned to a user who self signs up to the Developer portal.

Reply:
Fixed in https://github.com/wso2/docs-apim/pull/7842/commits/edc87080eaa5d2ace372481bc1b08590f28c92bd